stardog/admin.py

```diff
@@ -964,7 +928,10 @@ def set_options(self, options):
         Examples
             >>> db.set_options({'spatial.enabled': False})
         """
-        self.client.post(self.path + '/options', json=options)
+
+        # This should still be backwards compatible right?
```

Comment:
This should still be backwards compatible right? as in, adding a return value for a function that didn't have one explicit should not break stuff for people already using this function? im 99% sure it is fine but double checking just in case